src/Umbraco.Core/EmbeddedResources/Lang/tr.xml

```diff
@@ -692,7 +696,6 @@
     <key alias="off">Kapalı</key>
     <key alias="ok">Tamam</key>
     <key alias="open">Aç</key>
-    <key alias="options">Seçenekler</key>
```

Comment:
Bunu neden sildigimizi anlayamadim?

Reply:
Silmedim, diğer dillerdeki gibi <key alias="about">'in hemen üstüne taşıdım, bu sayede ilerde daha rahat kıyaslama yapma şansımız da olacak.